openbas-api/src/main/java/io/openbas/scheduler/jobs/InjectsExecutionJob.java

```diff
@@ -200,7 +200,7 @@ private void executeInject(ExecutableInject executableInject) {
               errorMsg -> finalStatus.getTraces().add(InjectStatusExecution.traceError(errorMsg)));
       finalStatus.setName(ExecutionStatus.ERROR);
       finalStatus.setTrackingSentDate(Instant.now());
-      finalStatus.setCommandsLines(injectUtils.getCommandsLinesFromInject(inject));
+      finalStatus.setPayloadOutput(injectUtils.getCommandsLinesFromInject(inject));
       injectStatusRepository.save(finalStatus);
```

Comment:
InjectUtils.getCommandsLinesFromInject maybe should be : InjectUtils.getPaylodOutput or getStatusPayloadOutput